airflow/hooks/presto_hook.py

```diff
@@ -80,7 +80,14 @@ def get_first(self, hql, parameters=None):
             return super(PrestoHook, self).get_first(
                 self._strip_sql(hql), parameters)
         except DatabaseError as e:
-            raise PrestoException(e[0]['message'])
+            if (hasattr(e, 'message')
```

Comment:
Would be nice the add a `self._handle_exception(e)` since the routine is copypasted